ui/StatusQ/include/StatusQ/statuswindow.h

```diff
@@ -25,16 +23,6 @@ class StatusWindow: public QQuickWindow
 private:
     void removeTitleBar();
     void showTitleBar();
-#ifdef Q_OS_WIN
-    void removeTitleBarWin();
-    void showTitleBarWin();
-#elif defined Q_OS_MACOS
-    void removeTitleBarMac();
-    void showTitleBarMac();
-#endif
```

Comment:
Must stay for the Windows/MacOS implementantions to work imo

Reply:
The implementations are still different depending on OS, it's just done simpler.